src/translations/en.json

```diff
@@ -160,14 +160,25 @@
   "nominalVoltage3": "Nominal Voltage 3",
   "nominalVoltage1KV": "Nominal Voltage 1 (kV)",
   "nominalVoltage2KV": "Nominal Voltage 2 (kV)",
+  "nominalVoltage3KV": "Nominal Voltage 3 (kV)",
   "voltageLevelId1": "Voltage level ID 1",
   "voltageLevelId2": "Voltage level ID 2",
+  "voltageLevelId3": "Voltage level ID 3",
   "terminal1Connected": "Connected 1",
   "terminal2Connected": "Connected 2",
+  "terminal3Connected": "Connected 3",
   "country1": "Country 1",
   "country2": "Country 2",
   "seriesResistance": "Series resistance (Ω)",
+  "seriesResistance1": "Series resistance 1 (Ω)",
+  "seriesResistance2": "Series resistance 2 (Ω)",
+  "seriesResistance3": "Series resistance 3 (Ω)",
   "seriesReactance": "Series reactance (Ω)",
+  "seriesReactance1": "Series reactance 1 (Ω)",
+  "seriesReactance2": "Series reactance 2 (Ω)",
+  "seriesReactance3": "Series reactance 3 (Ω)",
+  "shuntConductance": "Shunt conductance (μS)",
+  "shuntSusceptance": "Shunt susceptance (μS)",
```

Comment:
Is this in the scope of expert filter for hdvc line?

Reply:
yes it is not needed for hvdc it is for 3 windings tranformers I'll change the links

Reply:
In my opinion, it is better to not add more translate keys in explorer-app, we can profit this PR to clean old keys which similar to new keys, so we can clean voltageLevelIdXXX, terminalXXXConnected, countryXXX, seriesResistance, seriesReactance

Reply:
So I put the new translations in commons ui ?

Reply:
YES, keep this PR only for cleaning old keys after merge PR commons-ui

Reply:
alright